7_react/react1/week2/hw-react-week2/src/components/Container.jsx

```diff
@@ -0,0 +1,19 @@
+import { useState } from "react";
+
+import TaskList from "./TaskList";
+import FormTodo from "./FormTodo";
+
+const Container = () => {
+  const [list, setList] = useState([]); 
```

Comment:
again, avoid overly generic names like `list`. It doesn't really say much about what the variable actually contains and can make the code harder to understand. What could be a more descriptive name?